oviewer/prepare_draw.go

```diff
@@ -120,22 +120,20 @@ func (root *Root) setAlignConverter() {
 		maxWidths, addRight = m.maxColumnWidths(maxWidths, addRight, ln)
 	}
 
-	if !slices.Equal(m.alignConv.maxWidths, maxWidths) {
-		m.alignConv.orgWidths = m.columnWidths
-		m.alignConv.maxWidths = maxWidths
-		for n := len(m.alignConv.shrink); n < len(maxWidths)+1; n++ {
-			m.alignConv.shrink = append(m.alignConv.shrink, false)
-		}
-		for n := len(m.alignConv.rightAlign); n < len(maxWidths)+1; n++ {
-			m.alignConv.rightAlign = append(m.alignConv.rightAlign, false)
-		}
-		for i := 0; i < len(addRight); i++ {
-			if !m.alignConv.rightAlign[i] {
-				m.alignConv.rightAlign[i] = addRight[i] > 1
-			}
+	if slices.Equal(m.alignConv.maxWidths, maxWidths) {
+		return
+	}
+	m.alignConv.orgWidths = m.columnWidths
+	m.alignConv.maxWidths = maxWidths
+	for n := len(m.alignConv.columnAttrs); n < len(maxWidths)+1; n++ {
+		m.alignConv.columnAttrs = append(m.alignConv.columnAttrs, columnAttribute{})
+	}
```

Comment:
isn't equivalent to this ?

```suggestion
	m.alignConv.columnAttrs = make([]columnAttribute{}), len(maxWidths))
```

Reply:
This is different. Do not recreate the attributes.I'll add comments.